Source/WebCore/platform/graphics/avfoundation/objc/CDMSessionAVContentKeySession.mm

```diff
@@ -482,12 +495,7 @@ static bool isEqual(Uint8Array* data, ASCIILiteral literal)
         storageURL = [NSURL fileURLWithPath:storagePath.createNSString().get()];
     }
 
-    if ([PAL::getAVContentKeySessionClass() respondsToSelector:@selector(contentKeySessionWithKeySystem:storageDirectoryAtURL:)])
-        m_contentKeySession = [PAL::getAVContentKeySessionClass() contentKeySessionWithKeySystem:AVContentKeySystemFairPlayStreaming storageDirectoryAtURL:storageURL];
-    else
-        m_contentKeySession = adoptNS([PAL::allocAVContentKeySessionInstance() initWithStorageDirectoryAtURL:storageURL]);
-
-    [m_contentKeySession setDelegate:m_contentKeySessionDelegate.get() queue:m_delegateQueue->dispatchQueue()];
+    lazyInitialize(m_contentKeySession, createContentKeySession(storageURL));
```

Comment:
You could also pass storagePath instead of storageURL to make closer the if (m_contentSession) and the lazy init.